src/asm/opt.c

```diff
@@ -71,16 +78,31 @@ void opt_Parse(char *s)
 			error("Must specify exactly 4 characters for option 'g'\n");
 		break;
 
+	case 'f':
+		if (strlen(&s[1]) <= 2) {
+			int result;
+			unsigned int precision;
+
+			result = sscanf(&s[1], "%u", &precision);
```

Comment:
This should rather be `strtoul`, methinks.

Reply:
Sure, same for `fillByte` then (but with base 16).

Reply:
Actually, there is *one* difference, which is that `strtoul` allows arbitrary whitespace at the beginning. Maybe we can just parse the number manually instead—potentially sharing the code with the lexer, too?

Reply:
Can this be a separate small PR? Refactoring both `-Q` and `-p` to parse their numeric argument in whatever way (`strtoul`, `sscanf`, custom digit-by-digit; I'm fine with anything but using either of those std lib functions seems like the terse way to do it.)

Reply:
Yeah, sure.